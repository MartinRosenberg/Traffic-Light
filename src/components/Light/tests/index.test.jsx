import { shallow } from 'enzyme'
import React from 'react'

import { Light } from '..'

describe('Housing component', () => {
  [true, false].map(powered =>
    test(`should render light correctly when ${powered ? 'on' : 'off'}`, () => {
      const wrapper = shallow(<Light color={'red'} powered={powered}/>)
      expect(wrapper).toMatchSnapshot()
    })
  )
})
