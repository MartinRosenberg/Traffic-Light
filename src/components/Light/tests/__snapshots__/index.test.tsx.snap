// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Housing component should render light correctly when off 1`] = `
<div
  className="light"
/>
`;

exports[`Housing component should render light correctly when on 1`] = `
<div
  className="red light"
/>
`;
